tests/bootstrap/cdk_templates/LocalStackHostBootstrap/ClusterStack.json

```diff
@@ -0,0 +1,556 @@
+{
+  "Resources": {
+    "ResultsBucketA95A2103": {
+      "Type": "AWS::S3::Bucket",
+      "UpdateReplacePolicy": "Retain",
+      "DeletionPolicy": "Retain"
+    },
+    "Domain66AC69E0": {
+      "Type": "AWS::OpenSearchService::Domain",
+      "Properties": {
+        "ClusterConfig": {
+          "DedicatedMasterEnabled": false,
+          "InstanceCount": 1,
+          "InstanceType": "r5.large.search",
+          "ZoneAwarenessEnabled": false
+        },
+        "DomainEndpointOptions": {
+          "EnforceHTTPS": false,
+          "TLSSecurityPolicy": "Policy-Min-TLS-1-0-2019-07"
+        },
+        "DomainName": "domain-cf686810",
```

Comment:
I guess we don't need to transform these because the stack serves as one generated output 🤔 (just double-check)

Reply:
Yeah sort of. This is just a cache that prevents needless `cdk synth`. The domain name can be fixed for this.